src/02-common-principles.md

```diff
@@ -413,16 +413,21 @@ label, but must be included in file names (similarly to other key names).
 ## Units
 
 All units SHOULD be specified as per [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units)
-(abbreviated
-as SI, from the French Système international (d'unités)) and can be SI units or
-[SI derived units](https://en.wikipedia.org/wiki/SI_derived_unit).
-In case there are valid reasons to deviate from SI units or SI
-derived units, the units MUST be specified in the sidecar JSON file. In case
-data is expressed in SI units or SI derived units, the units MAY be specified in
-the sidecar JSON file. In case non-standard prefixes are added to SI or non-SI
-units, these non-standard prefixed units MUST be specified in the JSON file.
+(abbreviated as SI, from the French Système international (d'unités)) and can
+be SI units or [SI derived units](https://en.wikipedia.org/wiki/SI_derived_unit).
+In case there are valid reasons to deviate from SI units or SI derived units,
+the units MUST be specified in the sidecar JSON file.
+In case data is expressed in SI units or SI derived units, the units MAY be
+specified in the sidecar JSON file.
+In case non-standard prefixes are added to SI or non-SI units, these
+non-standard prefixed units MUST be specified in the JSON file.
 See [Appendix V](99-appendices/05-units.md) for a list of standard units and
 prefixes.
+Note also that for the *formatting* of SI units, the [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12)
+convention for encoding units is REQUIRED.
+More information about CMIXF can be found in [Appendix V](99-appendices/05-units.md)
+but to provide some examples, CMIXF formatting of "micro volts" would be `uV`,
+and of "kilobecquerel per milliliter" would be `kBq/mL`.
```

Comment:
I think we should also give an example of what would be an invalid CMIXF formatting.

Reply:
There is:

 1.  `uV` or `µV` are permissible, but NOT: `microV`, `µvolt`, `1e-6V`, etc.

a bit further below, I think that suffices, but feel free to make me do it if you feel strongly about it

Reply:
yeah I noticed once I read the whole thing. I think CMIXF comes out of the blue a bit. If there was an example to motivate why to use CMIXF it may help. But it's up to you.